packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalButtonLabelDictionary.st

```diff
@@ -0,0 +1,34 @@
+accessing
+globalButtonLabelDictionary
+
+	^ Dictionary
+		newFrom: {
+			'pauseAll' -> 'Pause All'.
+			'resumeAll' -> 'Resume All'.
+			'runAll' -> 'Run All'.
+			'openOverview' -> 'Open Overview'.
+			'toggleSounds' -> 'Toggle Sounds'.
+		
+			'createWatcher' -> 'Create Watcher'.
+			'pauseWatchers' -> 'Pause Watchers'.
+			'resumeSelectedWatchers' -> 'Resume Selected Watchers'.
+			'runSelectedWatchers' -> 'Run Selected Watchers'.
+			'editWatchers' -> 'Edit Watchers'.
+			'infoWatchers' -> 'Watcher Info'.
+			'resultsWatchers' -> 'Watcher Results'.
+			'removeSelectedWatchers' -> 'Remove Selected Watchers'.
+			
+			'createTravisStatus' -> 'Create Travis Status'.
+			'pauseTravisStatuses' -> 'Pause Travis Statuses'.
+			'resumeSelectedTravisStatuses' -> 'Resume Selected Travis Statuses'.
+			'editTravisStatuses' -> 'Edit Travis Statuses'.
+			'resultsTravisStatuses' -> 'Travis Statuses Info'.
+			'removeSelectedTravisStatuses' -> 'Remove Selected Travis Statuses'.
+			
+			'createGitHubAction' -> 'Create GitHub Action'.
+			'pauseGitHubActions' -> 'Pause GitHub Actions'.
+			'resumeSelectedGitHubActions' -> 'Resume Selected GitHub Actions'.
+			'editGitHubActions' -> 'Edit GitHub Actions'.
+			'resultsGitHubActions' -> 'GitHub Actions Info'.
+			'removeSelectedGitHubActions' -> 'Remove Selected GitHub Actions'
+		}
```

Comment:
```suggestion
accessing
buttonLabelDictionary

	^ Dictionary
		newFrom: {
			'pauseAll' -> 'Pause All'.
			'resumeAll' -> 'Resume All'.
			'runAll' -> 'Run All'.
			'openOverview' -> 'Open Overview'.
			'toggleSounds' -> 'Toggle Sounds'.
		
			'createWatcher' -> 'Create Watcher'.
			'pauseWatchers' -> 'Pause Watchers'.
			'resumeSelectedWatchers' -> 'Resume Selected Watchers'.
			'runSelectedWatchers' -> 'Run Selected Watchers'.
			'editWatchers' -> 'Edit Watchers'.
			'infoWatchers' -> 'Watcher Info'.
			'resultsWatchers' -> 'Watcher Results'.
			'removeSelectedWatchers' -> 'Remove Selected Watchers'.
			
			'createTravisStatus' -> 'Create Travis Status'.
			'pauseTravisStatuses' -> 'Pause Travis Statuses'.
			'resumeSelectedTravisStatuses' -> 'Resume Selected Travis Statuses'.
			'editTravisStatuses' -> 'Edit Travis Statuses'.
			'resultsTravisStatuses' -> 'Travis Statuses Info'.
			'removeSelectedTravisStatuses' -> 'Remove Selected Travis Statuses'.
			
			'createGitHubAction' -> 'Create GitHub Action'.
			'pauseGitHubActions' -> 'Pause GitHub Actions'.
			'resumeSelectedGitHubActions' -> 'Resume Selected GitHub Actions'.
			'editGitHubActions' -> 'Edit GitHub Actions'.
			'resultsGitHubActions' -> 'GitHub Actions Info'.
			'removeSelectedGitHubActions' -> 'Remove Selected GitHub Actions'
		}
```
The global buttons are just the first four, but i really like the idea of putting all button information into one dictionary (not one for each testing entity plus globals)

Could you fuse this dict with globalButtonFunctionalityDict, ... (we can do this together too)

Reply:
do you mean `globalFunctionalityDict` ? I can't find any methods called `globalButtonFunctionalityDict` (at least in the current dev branch, might be in a refactor branch)